sample/src_polymer/elements/scene-perception-pages/scene-perception-page.html

```diff
@@ -1,18 +1,229 @@
-<link rel="import" href="../elements.html">
-
+<link rel="import" href="../toggle-image-button/toggle-image-button.html">
+<link rel="import" href="../switch-image-button/switch-image-button.html">
```

Comment:
Why not import `elements.html`? We should keep only one sentence to import html.